lib/mods/replace-get-set.js

```diff
@@ -24,47 +24,62 @@ function isDataProperty(type, propertyName) {
       dataProperties.get(type).includes(propertyName);
 }
 
+/**
+ * @param {Node} node - the current ts-morph AST Node
+ *
+ * @return {boolean} whether the node is a global get
+ */
+function isGlobalGet(node) {
+  return nodeTypes.isPropertyAccessExpression(node) &&
+    !nodeTypes.isCallExpression(node.getParent()) &&
+    isGlobal(node.getExpression());
+}
+
+/**
+ * @param {Node} node - the current ts-morph AST Node
+ *
+ * @return {boolean} whether the node is a global set
+ */
+function isGlobalSet(node) {
+  return nodeTypes.isBinaryExpression(node) &&
+    node.getOperatorToken().getText() === '=' &&
+    nodeTypes.isPropertyAccessExpression(node.getLeft()) &&
+    isGlobal(node.getLeft().getExpression());
+}
+
 /**
  * Replaces getset calls in the current node with the Get-Originals version
  *
  * @param {Node} node - a ts-morph AST Node
  * @param {Object} imports - the current list of imports
  */
 function replaceGlobalGetSetCalls(node, imports) {
-  if (nodeTypes.isPropertyAccessExpression(node) &&
-      !nodeTypes.isCallExpression(node.getParent()) &&
-      isGlobal(node.getExpression())) {
-    const propertyName = node.getSymbol().getEscapedName();
-    const baseTypeName = node.getExpression().getType().getSymbol().getName();
+  let args; let type; let nodeToConsider;
```

Comment:
nodeToConsider -> propertyAccessNode, maybe

Reply:
The name nodeToConsider was meant to be somewhat forward thinking, since when I add window gets/sets they won't be property accesses, but I can change it for now